autoload/verilog_systemverilog.vim

```diff
@@ -599,6 +604,6 @@ function! verilog_systemverilog#VerilogErrorFormat(...)
     set errorformat+=UVM_%tATAL\ %f(%l)\ %m
   endif
 endfunction
-" }}}
+" 
```

Comment:
Please do not include this (and the equivalent at the top) in this Pull Request.